schemas/_common-schema.json

```diff
@@ -81,19 +81,11 @@
           "type": "string",
           "description": "User defined name of the view",
           "readOnly": false
-        }
-      },
-      "readOnly": true
-    },
-    "focus": {
-      "type": "object",
-      "description": "Focus properties",
-      "required": ["start_focused"],
-      "properties": {
-        "start_focused": {
+        },
+        "in_foreground": {
```

Comment:
If we don't plan to have this field on all events, I'd be in favor of having it only on the corresponding events schema.